### DISCUSSION TITLE
guide/drivers/local

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#103] [Announcements]
# guide/drivers/local

Root folder path The path of folder you wanted to mount. For example: Linux: /root; Windows: C:;

https://alist.nn.ci/guide/drivers/local.html

**Comment by TuRDMaN** [comment]
What is the difference between "mount path" and "root folder path"?

**Reply by xhofe** [reply]
- `mount path` is a virtual path in `AList`
- `root folder path` is the actual path on your machine

For example, your `mount path` is `/local` and your `root folder path` is `/home/username`: 
if you enter `https://xx.com/local`, it will show all the entries in the `/home/username`
